ZupIT/beagle-web-react
# Reorganizes imports

closes #146 
closes #143
**- What I did**
Removed styled components dependency from root. Altered React packages' same dependency from devDependencies to Dependencies
**- How I did it**

**- How to verify it**
Create new project and add beagle web react library, It should run and not ask for any additional dependency
**- Description for the changelog**
